tools/runAllTests.sh

```diff
@@ -121,7 +121,19 @@ if [ \( "$DO_ALL" -eq 1 \) -o \( "$DO_APP_PLUGINS" -eq 1 \) ]; then
 	if [ \( "$DO_COVERAGE" -eq 1 \) ]; then
 		REPORT_SWITCH="--coverage-php $REPORT_TMP/coverage-APP_PLUGINS.php"
 	fi
-	$phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH plugins
+	$phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH plugins/generic/lensGalley/tests
```

Comment:
I worry that listing these out manually will lead to forgotten entries and thus forgotten tests. If you can't specify a wildcard directly to `phpunit`, then maybe a shell script loop through available directories, or a `find -exec` call, would be better.

Reply:
I've never written a bash script or set up PHPUnit before. Can you recommend a solution?

Reply:
This line should do it:
`find plugins -name tests -maxdepth 3 -type d -exec $phpunit $DEBUG $TEST_CONF2 -v $REPORT_SWITCH "{}" ";"`

Reply:
cheers! :+1: